src/Raven.Studio/typescript/viewmodels/database/settings/connectionStrings.ts

```diff
@@ -166,21 +199,26 @@ class connectionStrings extends viewModelBase {
                 // sqlEtl
                 this.sqlEtlConnectionStringsNames(Object.keys(result.SqlConnectionStrings));
                 this.sqlEtlConnectionStringsNames(_.sortBy(this.sqlEtlConnectionStringsNames(), x => x.toUpperCase()));
+
+                // olapEtl
+                this.olapEtlConnectionStringsNames(Object.keys(result.OlapConnectionStrings));
+                this.olapEtlConnectionStringsNames(_.sortBy(this.olapEtlConnectionStringsNames(), x => x.toUpperCase()));
             });
     }
 
     confirmDelete(connectionStringName: string, connectionStringtype: Raven.Client.Documents.Operations.ConnectionStrings.ConnectionStringType) {
-        const stringType = connectionStringtype === "Raven" ? "RavenDB" : "SQL";
-        this.confirmationMessage("Are you sure?",
-            `Do you want to delete ${stringType} ETL connection string: <br><strong>${generalUtils.escapeHtml(connectionStringName)}</strong>`, {
-            buttons: ["Cancel", "Delete"],
-            html: true
-        })
+        const stringType = connectionStringtype === "Raven" ? "RavenDB" :  connectionStringtype === "Sql" ? "SQL" : "Olap";
```

Comment:
I've seen this login in some other place - I think it is time to move that to some static method with maps that .